tests/integration/tier1/set-locale/use_non_english_language.py

```diff
@@ -8,8 +8,8 @@ def test_use_non_english_language(shell):
     Ref bug: https://bugzilla.redhat.com/show_bug.cgi?id=2022854
     """
     # install Chinese language pack for CentOS-8 and Oracle Linux 8
-    os = platform.platform()
-    if "centos-8" in os or "oracle-8" in os:
+
+    if SYSTEM_RELEASE in ("centos-8.5", "centos-8.4", "oracle-8.7"):
```

Comment:
I would maybe turn the condition here into something like this
```suggestion
    if "centos-8" in SYSTEM_RELEASE or "oracle-8" in SYSTEM_RELEASE:
```
This way we do not have to care about the minor version of OL8 once this changes in the future.

Reply:
That makes sense, thanks!